src/mumble/AudioOutputSample.cpp

```diff
@@ -202,7 +205,8 @@ QString AudioOutputSample::browseForSndfile(QString defaultpath) {
 	return file;
 }
 
-bool AudioOutputSample::prepareSampleBuffer(unsigned int snum) {
+bool AudioOutputSample::prepareSampleBuffer(unsigned int frameNumber) {
+	unsigned int snum = frameNumber * sfHandle->channels();
```

Comment:
While we're working on this part of the code anyways: Could you give a proper name to this variable, please? I assume this should be the number of samples? In that case I'd suggest `sampleCount`

Reply:
`snum` is a name used in old code. Actually at first it took me some time to figure it out what it means... I'll change them.